Makefile

```diff
@@ -107,7 +107,7 @@ p4-build: p4src/main.p4
 		p4src/main.p4
 	@echo "*** P4 program compiled successfully! Output files are in p4src/build"
 
-p4-test:
+p4-test: _docker_build
```

Comment:
Not required. `_docker_build` is called by `make deps`

Reply:
But `make deps` is never used elsewhere and also not mentioned in the exercises.

Reply:
It is here: https://github.com/opennetworkinglab/ngsdn-tutorial/tree/princeton19#download--upgrade-dependencies

Reply:
Remove target, should be done as part of provisioning